graspy/plot/plot.py

```diff
@@ -401,11 +438,12 @@ def gridplot(
         plot.ax.axis("off")
         plot.ax.invert_yaxis()
         if title is not None:
-            if inner_hier_labels is not None:
-                pad = 1.5 * font_scale + 1 * hier_label_fontsize + 30
-            else:
-                pad = 1.5 * font_scale + 15
-            plt.title(title, pad=pad)
+            if title_pad is None:
+                if inner_hier_labels is not None:
+                    title_pad = 1.5 * font_scale + 1 * hier_label_fontsize + 30
+                else:
+                    title_pad = 1.5 * font_scale + 15
+            plt.title(title, pad=title_pad)
```

Comment:
the title_pad is not big enough if both inner and outer hier_labels are passed

Reply:
make a PR into this branch? Automatic settings that always work are tricky though. In the meantime the `title_pad` kwarg is exposed now